### PR TITLE
feat[WEB-2]: build a container layout for the home page

### PR DESCRIPTION
**Pull request by MyHKo**
Build a complete container layout for the home page

### CODE REVIEW

File: src/pages/Home/components/HeroSection/hero.module.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.container {
+  height: 700px;
+  background-color: #007aff;
+  position: sticky;
+  top: 0;
+  display: flex;
+  align-items: center;
+}
```
Comment (AndrZakhara) [inline src/pages/Home/components/HeroSection/hero.module.scss:8]: add empty line in the end of file

File: src/pages/Home/components/PrototypeSection/prototype.module.scss [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.container {
+  height: 700px;
+  background-color: red;
+  position: sticky;
+  top: 0;
+}
```
Comment (AndrZakhara) [inline src/pages/Home/components/PrototypeSection/prototype.module.scss:6]: the same, empty line

File: src/pages/Home/components/UpdatesSection/updates.module.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.container {
+  height: 700px;
+  background-color: green;
+  position: relative;
+}
```
Comment (AndrZakhara) [inline src/pages/Home/components/UpdatesSection/updates.module.scss:5]: add empty line in the end of file

File: src/pages/Home/index.jsx [hunk 4]
```diff
@@ -1,9 +1,16 @@
...
+import HeroSection from '@pages/Home/components/HeroSection/index.jsx'
+import PrototypeSection from '@pages/Home/components/PrototypeSection/index.jsx'
+import UpdatesSection from '@pages/Home/components/UpdatesSection/index.jsx'
+import DiscoverSection from '@pages/Home/components/DiscoverSection/index.jsx'
 
 export default function Home() {
   return (
     <div className={styles.container}>
-      <div>Home</div>
+      <HeroSection />
```
Comment (AndrZakhara) [inline src/pages/Home/index.jsx:10]: use shorter names: Hero, Prototype, Updates. etc

File: src/pages/Home/index.jsx [hunk 5]
```diff
@@ -1,9 +1,16 @@
 import styles from './home.module.scss'
+import HeroSection from '@pages/Home/components/HeroSection/index.jsx'
```
Comment (AndrZakhara) [inline src/pages/Home/index.jsx:2]: U can use this path in import, index.jsx isn't needed: 
import HeroSection from '@pages/Home/components/HeroSection

File: src/styles/_reset.scss [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+  content: '';
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
```
Comment (AndrZakhara) [inline src/styles/_reset.scss:43]: empty line

**Review by AndrZakhara — APPROVED** [review]

**Review by sofiiavoitkova — APPROVED** [review]

**Review by GarnaA — APPROVED** [review]

**Pull request merged**